include/seqan3/range/view/translate.hpp

```diff
@@ -353,6 +434,19 @@ class view_translate_single
 
     //!\brief Implicit conversion to container types.
     template <RandomAccessContainer container_type>
+    operator container_type()
+    //!\cond
+        requires std::is_same_v<aa27, value_type_t<container_type>>
+    //!\endcond
+    {
+        container_type ret;
+        ret.resize(size());
+        std::copy(cbegin(), cend(), ret.begin());
+        return ret;
+    }
+
+    //!\overload
+    template <RandomAccessContainer container_type>
     operator container_type() const
     //!\cond
         requires std::is_same_v<aa27, value_type_t<container_type>>
```

Comment:
Please remove both of the conversion operators, they are not part of View interfaces anymore. We have #1033 now.